Moderation_Matrix.md

```diff
@@ -0,0 +1,59 @@
+# Moderation Matrix
+This matrix correlates with the [Community Rules](https://www.theodinproject.com/community_rules) that are required to be followed in order to participate in the community. Some rules have different levels of severity, so you may see references to them under different numbers of zaps. 
+
+This matrix is not all encompassing and zaps are given at the discretion of the moderator team.
+
+## 0 Zaps (Heads Up)
+- Dog piling/short circuiting a teaching moment
+- Asking about or encouraging the use of Windows
+- "Chat bombing", aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message) 
+- Discussing mental health issues in depth
+
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project 
+-Mild unprofessionalism:
+  - Drug related conversation 
+  - Treating others on the server disrespectfully
+  - Mental health issues discussion 
+  - Relationship advice
+- A second infraction of any 0 Zap issues above
+
+## 2 Zaps (WARNING)
+Medium severity violations such as: 
+- Suggesting piracy 
```

Comment:
Should "suggesting piracy" really be separate from "facilitating it"? Not a hill I'll die on, but am curious if it wouldn't be better to have any form of piracy at a single level rather than split out like this.

Reply:
Definitely true. Mainly wanted to see if we wanted to keep them as separate offenses or just lump them together as a general "no piracy" rule. But again, not a hill I'll die on and I'm good with things as-is.